textattack/commands/augment.py

```diff
@@ -28,62 +28,101 @@ def run(self, args):
+                transformations_per_example=args.transformations_per_example,
+            )
 
-        output_rows = []
-        for row in tqdm.tqdm(rows, desc="Augmenting rows"):
-            text_input = row[args.input_column]
-            if not args.exclude_original:
-                output_rows.append(row)
-            for augmentation in augmenter.augment(text_input):
-                augmented_row = row.copy()
-                augmented_row[args.input_column] = augmentation
-                output_rows.append(augmented_row)
-        # Print to file.
-        with open(args.outfile, "w") as outfile:
-            csv_writer = csv.writer(
-                outfile, delimiter=",", quotechar='"', quoting=csv.QUOTE_MINIMAL
+            output_rows = []
+            for row in tqdm.tqdm(rows, desc="Augmenting rows"):
+                text_input = row[args.input_column]
+                if not args.exclude_original:
+                    output_rows.append(row)
+                for augmentation in augmenter.augment(text_input):
+                    augmented_row = row.copy()
+                    augmented_row[args.input_column] = augmentation
+                    output_rows.append(augmented_row)
+            # Print to file.
+            with open(args.outfile, "w") as outfile:
+                csv_writer = csv.writer(
+                    outfile, delimiter=",", quotechar='"', quoting=csv.QUOTE_MINIMAL
+                )
+                # Write header.
+                csv_writer.writerow(output_rows[0].keys())
+                # Write rows.
+                for row in output_rows:
+                    csv_writer.writerow(row.values())
+            textattack.shared.logger.info(
+                f"Wrote {len(output_rows)} augmentations to {args.outfile} in {time.time() - start_time}s."
             )
-            # Write header.
-            csv_writer.writerow(output_rows[0].keys())
-            # Write rows.
-            for row in output_rows:
-                csv_writer.writerow(row.values())
-        textattack.shared.logger.info(
-            f"Wrote {len(output_rows)} augmentations to {args.outfile} in {time.time() - start_time}s."
-        )
+        else:
+            print("\nRunning in interactive mode")
+            print("----------------------------")
+            while True:
+                print(
+                    'Enter a sentence to augment, "q" to quit, "c" to change arguments:'
+                )
+                text = input()
+
+                if text == "q":
+                    break
+
+                if text == "c":
+                    print("recipe: ", end=" ")
+                    args.recipe = input()
+                    print("pct-words-to-swap: ", end=" ")
+                    args.pct_words_to_swap = float(input())
+                    print("transformations-per-example: ", end=" ")
+                    args.transformations_per_example = int(input())
+                    continue
```

Comment:
[1] this is nifty. Allowing users to change the arguments without restarting the program is a really good idea. Let's try to make it look a little more professional:
- at the beginning of this if statement, let's print an orienting message for the user. maybe: `print("Changing augmenter arguments...")`, or something like that
- then, can you print a tab before all of the words typed as arguments? like: `print("\trecipe: ")`

[2] rename the prompts to be a little more user-friendly: 
- `"recipe" -> "Augmentation recipe name"`
- `"pct-words-to-swap" -> "Percentage of words to swap"`
- `"transformations-per-example" -> "Transformations per input example"` 

[3] btw, do you need the `end=" "` argument provided to these print statements? I'd expect the thing to look good as-is (since you added a space to the end of the printed string already)

Reply:
Thanks for the suggestions! I implemented these changes along with some others for a better visualization, like command spacings when they are printed out. 

When asking for `"Augmentation recipe name"`, I also included a condition in which if users type in "r", the list of available recipes will be printed. It could serve as a nice reminder (though there are only four of them). Do you think this could be helpful?